graph paper.tex

```diff
@@ -1,6 +1,6 @@
 \documentclass{article}
 
-\usepackage[pattern=hexup]{gridpapers}
+\usepackage[pattern=iso,minorcolor=cornflower!30]{gridpapers}
 
 \begin{document}  
 \thispagestyle{empty}
```

Comment:
Does this file need to be touched for this commit?

Reply:
No, I’m being super sloppy